decanter_ai_sdk/client.py

```diff
@@ -121,7 +123,7 @@ def train_iid(
         seed: int = 1180,
         timeseries_value: List[Dict[str, Any]] = [],
         holdout_percentage: int = 10,
-        missing_value_settings: Dict[str, Any] = {},
+        missing_value_settings: Dict[str, Missing_Value_Handling] = {},
```

Comment:
Should not be a mixture of snake and camel case. Camel case is fine

Reply:
OMG my bad!!